builtin/logical/database/path_roles_test.go

```diff
@@ -344,6 +345,19 @@ func TestBackend_StaticRole_Config(t *testing.T) {
 			path: "disallowed-role",
 			err:  errors.New("\"disallowed-role\" is not an allowed role"),
 		},
+		"fails to parse cronSpec with seconds": {
+			account: map[string]interface{}{
+				"username":          dbUser,
+				"rotation_schedule": "*/10 * * * * *",
+			},
+			path: "plugin-role-test-1",
+			expected: map[string]interface{}{
+				"username":          dbUser,
+				"rotation_schedule": "*/10 * * * * *",
+			},
+			scheduleOptions: scheduleOptionsDefault,
```

Comment:
```suggestion

```

Reply:
We should be able to remove `scheduleOptions` since we aren't using the override in the setup for this test. And we expect an error so `expected` isn't needed either.

Reply:
And we should be able to remove `scheduleOptions` from the inline struct definition above.

Reply:
Yep, that's right. Thank you. Just updated it 👍